integ-test/src/test/java/org/opensearch/sql/sql/SimpleQueryStringIT.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.sql;
+
+import java.io.IOException;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.opensearch.sql.legacy.SQLIntegTestCase;
+
+public class SimpleQueryStringIT extends SQLIntegTestCase {
+  @Override
+  public void init() throws IOException {
+    loadIndex(Index.BEER);
+  }
+
+  /*
+  The 'beer.stackexchange' index is a dump of beer.stackexchange.com converted to the format which might be ingested by OpenSearch.
+  This is a forum like StackOverflow with questions about beer brewing. The dump contains both questions, answers and comments.
+  The reference query is:
+    select count(Id) from beer.stackexchange where simple_query_string(["Tags" ^ 1.5, Title, `Body` 4.2], 'taste') and Tags like '% % %' and Title like '%';
+  It filters out empty `Tags` and `Title`.
+  */
+
+  @Test
+  public void test1() throws IOException {
+    String query = "SELECT count(*) FROM "
+        + Index.BEER.getName() + " WHERE simple_query_string([\\\"Tags\\\" ^ 1.5, Title, `Body` 4.2], 'taste')";
+    var result = new JSONObject(executeQuery(query, "jdbc"));
+    assertNotEquals(0, result.getInt("total"));
+  }
+
+  @Test
+  public void verify_wildcard_test() throws IOException {
+    String query1 = "SELECT count(*) FROM "
+        + Index.BEER.getName() + " WHERE simple_query_string(['Tags'], 'taste')";
+    var result1 = new JSONObject(executeQuery(query1, "jdbc"));
+    String query2 = "SELECT count(*) FROM "
+        + Index.BEER.getName() + " WHERE simple_query_string(['T*'], 'taste')";
```

Comment:
Couldn't `TestConstants.TEST_INDEX_BEER` be used here?

Reply:
Fixed in c1a91fc.